### PR TITLE
Reorganized directory structure in compliance with Swift Package guid…

### PR DESCRIPTION
**Pull request by KedarDhere**
# Proposed Changes

This PR addresses the restructuring of our current file system to align with [Swift Package guidelines](https://developer.apple.com/documentation/xcode/creating-a-standalone-swift-package-with-xcode). By doing this, we ensure that our project is organized in a manner consistent with Swift best practices, leading to better maintainability and clarity.

# Jira Stories

- [FM-9](https://oregonstate-innovationlab.atlassian.net/browse/FM-9?atlOrigin=eyJpIjoiNDliN2M0NzQyODZlNDUzN2I0MTgzZmJkNjBhYzljNjIiLCJwIjoiaiJ9)

# Instructions to review

1. Begin by cloning the branch to your local machine.
2. Navigate to the restructured files and inspect the layout. Compare it to the [Swift Package guidelines](https://developer.apple.com/documentation/xcode/creating-a-standalone-swift-package-with-xcode) to ensure compliance.
3. Review the changes made in the commit diffs to understand the exact adjustments.
4. Leave any comments or feedback regarding improvements or concerns.

### CODE REVIEW

**Comment by wbraynen** [comment]
As per Swift conventions, please make sure all your filenames begin with a capital letter (and contain a type by the same name, the name of which should also begin with a capital letter).

File: Tests/FlowMagic-SDKTests/FlowMagic_SDKTests.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import XCTest
+@testable import FlowMagic_SDK
+
+final class FlowMagicSDKTests: XCTestCase {
+    func testExample() throws {
+
+    }
+}
```
Comment (wbraynen) [inline Tests/FlowMagic-SDKTests/FlowMagic_SDKTests.swift:8]: Why add this file?  The only method it contains is a no-op.

Comment (wbraynen) [reply Tests/FlowMagic-SDKTests/FlowMagic_SDKTests.swift:8]: Also, does this mean you have no tests?  If so, add some.

Comment (KedarDhere) [reply Tests/FlowMagic-SDKTests/FlowMagic_SDKTests.swift:8]: Test cases are added in a separate [PR](https://github.com/KedarDhere/FlowMagic-SDK/pull/7). This PR includes only file organization-related changes.

**Comment by souvikdas2411** [comment]
Hello! I have slacked the screenshots.

**Review by souvikdas2411 — APPROVED** [review]

**Pull request merged**